thunderfish/eodanalysis.py

```diff
@@ -72,20 +69,16 @@ def eod_analysis_plot(time, mean_eod, std_eod, ax):
     :param std_eod:             (1-D array) containing the data of the std eod.
     :param ax:                  (axis for plot) empty axis that is filled with content in the function.
     """
-    u_std = [mean_eod[i] + std_eod[i] for i in np.arange(len(mean_eod))]
-    l_std = [mean_eod[i] - std_eod[i] for i in np.arange(len(mean_eod))]
+    u_std = [mean_eod[i] + std_eod[i] for i in range(len(mean_eod))]
+    l_std = [mean_eod[i] - std_eod[i] for i in range(len(mean_eod))]
     ax.plot(time, mean_eod, lw=2, color='firebrick', alpha=0.7, label='mean EOD')
     ax.fill_between(time, u_std, l_std, color='grey', alpha=0.3)
```

Comment:
Wih numpy arrays this is just:
ax.fill_between(time, mean_eod+std_eod, mean_eod-std_eod, color='grey', alpha=0.3)